yarn.lock

```diff
@@ -62,6 +62,14 @@
     "@opentelemetry/api" "^0.6.1"
     tslib "^2.0.0"
 
+"@azure/core-auth@^1.1.4":
```

Comment:
Is there only way to edit yarn.lock for version update?

Reply:
I can revert my last commit and update only url-parse in yarn.lock (first commit)

Reply:
will the `resolutions` under root\package.json work instead? I am concerning about editing yarn.lock, usually if we run `yarn install` it gets reverted to the original state.

Reply:
`resolutions` should not be necessary in this case, and ideally they are not used unless absolutely necessary. The (only) root dependency for this component is `@azure/batch`, which lists `url-parse "^1.4.7"` with caret `^` version, so any minor releases are acceptable, including 1.5.1. I also edited yarn.lock by removing the entry for url-parse and then running `yarn install`, which re-added url-parse with its newest acceptable version. Since I ran `yarn install` to get this update, running `yarn install` in the future will not revert it to 1.4.7 unless `@azure/batch` changes its dependency to be ~1.4.7 (patch releases acceptable), 1.4.7 (specifically this version), or similar.